File: C/prime_no_do_while.cpp
#include<stdio.h>
int main()
{
	int num;
	int counter=2;
	int rem=0;
	printf("enter num");
	scanf("%d",&num);
	do
	{
		rem=num%counter;
		if(rem==0)
		{
			printf("not a prime no");
		}
		counter=counter+1;
		
	}
	while(counter<num);
	    if(num==counter)
	{
		printf("it is a prime no");
		
	}
	
}

File: c_sharp/Prime_No_DoWhile.cs
using System;

namespace Prime_Num
{
    internal class Program
    {
        static void Main(string[]args)
        {
            int num;
            int counter = 2;
            int flag = 0;
            Console.Write("Enter Number: ");
            num = Convert.ToInt32(Console.ReadLine());
            do
            {
                if (num % counter == 0)
                {
                    flag = 1;
                    break;
                }
                counter++;
            } while (counter < num);
            if (flag == 0)
            {
                Console.WriteLine("It is a Prime number");
            }
            else
            {
                Console.WriteLine("It is Not a Prime Number");
            }
            Console.ReadKey();
        }
    }
}
